src/popup.js

```
@@ -37,3 +37,16 @@ document.addEventListener('DOMContentLoaded', () => {
         }, 2300); // 2s bekle + 0.3s fadeout
     }
 });
+
+if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+    document.body.classList.add('dark-mode');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)